lib/Mojo/IOLoop/ReadWriteProcess.pm

```diff
@@ -32,7 +33,7 @@ use Exporter 'import';
 use constant DEBUG => $ENV{MOJO_PROCESS_DEBUG};
 
 has [
-  qw(kill_sleeptime sleeptime_during_kill),
+  qw(kill_sleeptime sleeptime_during_kill patience_during_kill),
```

Comment:
If you want to introduce another parameter please include in documentation. "patience" and "sleeptime" are IMHO ambiguous in understanding. How about "interval_time" or "total_sleeptime_during_kill"?

Reply:
I suppose `total_sleeptime_during_kill` makes sense.